src/content/resources/link-token-contracts.mdx

```diff
@@ -402,9 +400,9 @@ Testnet LINK is available at [faucets.chain.link](https://faucets.chain.link/bas
 
 ### BASE Goerli testnet
 
-Testnet ETH is used to pay for transactions on BASE.
+Testnet ETH is used to pay for transactions on BASE. Testnet ETH is available from one of the [BASE Network Faucets](https://docs.base.org/tools/network-faucets).
```

Comment:
Base Goerli is deprecated. The Base testnet migration from Goerli to Sepolia has been finalized on 2/9/2024.